### PR TITLE
Update template for newer options

### LINKED ISSUE #7: UMASK setting in `/etc/sysconfig/sysstat`
EL9 includes a `UMASK` setting in `/etc/sysconfig/sysstat`:

```
# The sa1 and sa2 scripts generate system activity data and report files in
# the /var/log/sa directory. By default the files are created with umask 0022
# and are therefore readable for all users. Change this variable to restrict
# the permissions on the files (e.g. use 0027 to adhere to more strict
# security standards).
UMASK=0022
```

Can this module be extended to offer this flag and set the requested mode on `/var/log/sa`?

Older versions of sysstat will ignore the flag, but fixing up `/var/log/sa` would at least provide some level of restriction.

### PR DESCRIPTION
**Pull request by jcpunk**
This adds a few options present in newer versions of sysstat.

Fixes #7

### CODE REVIEW

File: manifests/init.pp [hunk 1]
```diff
@@ -111,13 +114,22 @@
...
+          sa_umask        => $sa_umask,
+          zip             => $zip,
         }),
       }
 
+      file { $sa_dir:
+        ensure => directory,
+        owner  => 'root',
+        group  => 'root',
+      }
```
Comment (mattqm) [inline manifests/init.pp:131]: I think this will work on AIX, but I don't have a system to test on

File: templates/sysconfig.epp [hunk 2]
```diff
@@ -15,6 +15,21 @@ COMPRESSAFTER=<%= $compressafter %>
 # which are used for the generation of log files.
 SADC_OPTIONS="<%= $sadc_options %>"
 
+# Directory where sa and sar files are saved. The directory must exist.
+SA_DIR=<%= $sa_dir %>
+
```
Comment (mattqm) [inline templates/sysconfig.epp:20]: But your code creates it, so maybe change the commentary to 'The directory will be created if it does not exist'.

File: manifests/init.pp (hunk 1 above)
Comment (jcpunk) [reply manifests/init.pp:131]: I do not have access to AIX either....

File: templates/sysconfig.epp (hunk 2 above)
Comment (jcpunk) [reply templates/sysconfig.epp:20]: I was trying to preserve the comment from the "un-puppetized" code.  The sysstat tools wont make the directory if it is missing.  If someone disables puppet and changes this value, they'll need to make the directory.

Comment (mattqm) [reply templates/sysconfig.epp:20]: sorry, bit of a brain freeze on my part - was thinking you were updating the read me - all good

File: manifests/init.pp (hunk 1 above)
Comment (mattqm) [reply manifests/init.pp:131]: oh well, I think it's ok from what I googled, I suppose we'll find out sooner or later if it's not.

**Comment by mattqm** [comment]
thanks for the PR

**Comment by jcpunk** [comment]
:) awesome, any chance for a new release on the forge?

**Comment by mattqm** [comment]
Done ;-)

**Pull request merged**